### ISSUE TITLE
ES modules to CommonJS conversion transparent in sourcemaps?

### ISSUE THREAD

**Issue by mightyiam** [s-panferov/awesome-typescript-loader#403]
The following is a screenshot from Chrome Devtools debugger, where import names are altered:
![screenshot_2017-03-27_11-06-16](https://cloud.githubusercontent.com/assets/635591/24346530/6a8b4460-12dd-11e7-884f-492d45b6f2e2.png)

`dom_1` is originally `dom`, `xstream_1` is originally `xstream` and so on.

I guess that's how TypeScript handles converting ES modules to CommonJS.

Isn't this supposed to be transparent in the source maps, though?

I must be doing something wrong. Here is [my current setup](https://github.com/mightyiam/cyclejs-components/tree/0bcd747e3bce5eeb233bb88e03cab463377ded34).

**Comment by abirmingham** [comment]
I'm experiencing the same issue. Were you able to resolve it?

**Comment by mightyiam** [comment]
Nope. I don't even know whether TypeScript does it or webpack does it.

**Comment by abirmingham** [comment]
I was able to track down [this](https://github.com/babel/babel/issues/2597) issue, which is similar except in Babel. The conclusion seems to be that there is a shortcoming in symbol sourcemapping in chromium and/or the sourcemap spec itself. I would imagine that TypeScript is subject to to the same shortcoming when emitting ES5.

**Comment by mightyiam** [comment]
I'd love to get a link to the issue with source maps or the implementations of it.
